cruise-control/src/yaml/endpoints/load.yaml

```diff
@@ -0,0 +1,65 @@
+LoadEndpoint:
+  get:
+    operationId: load
+    summary: Get the cluster load once Cruise Control Load Monitor is running.
+    parameters:
+      - name: start
+        in: query
+        description: Start time of the cluster load
+        schema:
+          type: long
+          format: int64
+          # Default is time of earliest valid window
+      - name: end
+        in: query
+        description: End time of the cluster load
+        schema:
+          type: long
+          format: int64
+          # Default is current system time
+      - name: time
+        in: query
+        description: End time of the clutser load
+        schema:
+          type: long
+          format: int64
+        # Default is current system time, mutually exclusive from end parameter
+      - name: allow_capacity_estimation
+        in: query
+        description: Whether to allow capacity estimation when cruise-control is unable to obtain all per-broker capacity information
+        schema:
+          type: boolean
+          default: true
+      - name: json
+        in: query
+        description: Whether to return in JSON format or not
+        schema:
+          type: boolean
+          default: false
+      - name: populate_disk_info
+        in: query
+        description: Whether show the load of each disk of broker
+        schema:
+          type: boolean
+          default: false
+    responses:
+      '200':
+        description: |-
```

Comment:
Could you keep the description on one line i.e. remove `|-`

Reply:
Done.